articles/why-and-how-zk-snark-works/3_non_interactive_zero-knowledge_of_a_polynomial.md

```diff
@@ -0,0 +1,260 @@
+---
+Title: Non interactive Zero-knowledge of a polynomial
+Status: Done
+Level: "3"
+---
+# Chapter 3 - Non-Interactive Zero Knowledge of a Polynomial
+
+## 3.1. Proving Knowledge of a Polynomial
+
+We start with a problem of proving the knowledge of a polynomial and make our way to a generic approach. We will discover many other
+properties of polynomials along the way.
+
+The discussion so far has focused on a weak notion of a proof, where parties have to trust each other because there are no measures yet
+to enforce the rules of the protocol. For example, the prover is not required to know a polynomial, and he can use any other means
```

Comment:
<img width="408" alt="Screenshot 2024-01-23 at 16 53 59" src="https://github.com/sota-zk-lab/zkp-documents/assets/33181397/3e3dc05e-83b2-4309-996c-7d0c63bb9553">
Only write the key idea, summarize it